ethcore/src/engines/authority_round.rs

```diff
@@ -83,6 +83,8 @@ pub struct AuthorityRound {
 	client: RwLock<Option<Weak<EngineClient>>>,
 	signer: EngineSigner,
 	validators: Box<ValidatorSet + Send + Sync>,
+	/// Is this Engine just for testing (prevents step calibration).
+	test: bool,
```

Comment:
Would be better to name it according to what it does. E.g. `disable_step_calibration`. Or, even better, invert it and name as `calibrate_step`